src/cryptoadvance/specter/static/helpers.js

```diff
@@ -1,7 +1,22 @@
+import { capitalize, numberWithCommas } from './helper-modules/formatting.js'
+import { copyText, send_request, showError, showNotification, wait } from './helper-modules/common.js'
+import { toggleMobileNav } from './helper-modules/mobile.js'
+
+// The scope of functions inside a module is not global, putting them on the window object makes them accessible outside of the module. 
```

Comment:
Thanks for the comment!  That makes it much clearer.

Reply:
Took me days to figure it out ...

Reply:
> Took me days to figure it out ...

Thank you for your work! This is a great step to have more powerful javascript functions (and maybe even classes and more) and test them properly.